src/rpc/common/Concepts.h

```diff
@@ -50,7 +50,7 @@ concept Requirement = requires(T a) {
  */
 // clang-format off
 template <typename T>
-concept CoroutineProcess = requires(T a, typename T::Input in, typename T::Output out, boost::asio::yield_context* y) {
+concept CoroutineProcess = requires(T a, typename T::Input in, typename T::Output out, boost::asio::yield_context& y) {
```

Comment:
I feel reference is more friendly when we use it in handler. try to hide pointer thing inside framework